### ISSUE TITLE
Upgrading to 1.3.X removes AESCBCEncryptionSecret

### ISSUE THREAD

**Issue by mrwulf** [budimanjojo/talhelper#81]
Per the [release docs](https://github.com/siderolabs/talos/releases/tag/v1.3.0), the AESCBC key should remain on clusters that already have it configured (so they can decrypt existing secrets).

Changing the version in talconfig.yaml, running genconfig, and diff (talosctl apply-config --dry-run) yields:
```
-               ClusterAESCBCEncryptionSecret:    "<REDACTED FOR THE INTERNET>",
+               ClusterAESCBCEncryptionSecret:    "",
```

**Comment by budimanjojo** [comment]
Thanks! It's the same at my end. Not sure how should I make sure the `aescbc` key should be there when it needs to be? Maybe I should check if the key is in the `talsecret` file? So if `secrets.aescbcencryptionsecret` is inside `talsecret.yaml` file, then the generated file should have it too?

**Comment by mrwulf** [comment]
Yeah- that seems like it makes sense. Talos will prioritize the secretbox key if both exist so there's no real risk.

**Comment by budimanjojo** [comment]
Fixed in v1.5.7
